### PR TITLE
Improvement in entity audit documentation

### PR DESCRIPTION
**Pull request by RTardelliGarcia**
Update the documentation due to the last changes in EntityAudit improvements

### CODE REVIEW

File: developers/backend-api/entity-audit.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # Entity Audit
 
-An **Entity Audit** represents a write operation on a **Live** entity, that was captured by a Hibernate Listener, and transformed into a new entity called `EntityAudit`. This new entity stores details of the operation and the new state of the entity in the system.
+An **Entity Audit** represents a write operation on a **Live** entity, that was listened by a Hibernate Listener, and transformed into a new entity called `EntityAudit`. This new entity stores details of the operation and the new state of the entity in the system.
```
Comment (hugomcs) [inline developers/backend-api/entity-audit.md:3]: "captured" was probably better. Listened by a listener doesn't sound well.

File: developers/backend-api/entity-audit.md [hunk 2]
```diff
@@ -18,6 +18,19 @@ There are some rules to an entity to be available for auditing. The `EntityAudit
 
 {% hint style="warning" %} **Delete operations do not store the complete content** of the entity due to a technical reason. In delete operations the Hibernate Listener can no longer have access to a relationship of the target entity. Another point is that this operation do not change the last version of the content being deleted and therefore would be a resource waste. {% endhint %}
 
+## Plugin auditable entities
+
+Starting in version 3.36, **Live** provides new API entry that permits plugins to register their own __Hibernate Session Factory__ into Live to expand the auditing features to the plugin's entities. Live will use the plugin's __Hibernate Session Factory__ to register a `EntityAuditDBListener` with all necessary `TypeAdapters` and used it to search for `EntityAudits` entities in the auditing endpoint (/rest/entity-audit). It is also possible to extend the listener in order to customize the common rules and add more skip conditions to avoid auditing a write operation. With that in hand the plugin can register the listener by itself in its _Hibernate Session Factory_ as this snippet bellow:
```
Comment (hugomcs) [inline developers/backend-api/entity-audit.md:23]: in version 3.36.0

File: developers/backend-api/entity-audit.md [hunk 3]
```diff
@@ -97,4 +110,6 @@ This endpoint retrieves the entity content from an `EntityAudit`. It is used to
 
 If needed, the user can use the result of this method to roll back parts or all of the current version of an entity by calling the save method on the related resource class.
 
-{% hint style="warning" %} Getting an EntityAudit that represents a delete operation will return a BAD_REQUEST. As explained before, EntityAudits of delete operations do not hold an entity content {% endhint %}
\ No newline at end of file
+{% hint style="warning" %} Getting an EntityAudit that represents a delete operation will return a BAD_REQUEST. As explained before, EntityAudits of delete operations do not hold an entity content {% endhint %}
+
+{% hint style="info" %} A change was made in the EntityAudits returned by the listing endpoints (List by type and List by type and id). That change replaces the `author` field from a structure like `{"id":1, "name":"someone"}` to a simple integer representing the id. This was necessary to keep the model of EntityAudit from Live and the plugins the same as the plugins do not store users information to fill in the value.
```
Comment (hugomcs) [inline developers/backend-api/entity-audit.md:115]: Please include the version when the change was made (3.36.0).

File: developers/backend-api/entity-audit.md (hunk 2 above)
Comment (michelpf13) [inline developers/backend-api/entity-audit.md:23]: As you put a snippet of the `Listener` implementation, I think it's worth doing the same for the `API` call. A snipper with `SessionFactory` and basic `EntityAuditOptions`. Maybe briefly explain `EntityAuditOptions` and its attributes.

Comment (RTardelliGarcia) [reply developers/backend-api/entity-audit.md]: Right. Check it out to see if it is ok

**Pull request merged**